lib/plugins/bed.js

```diff
@@ -10,35 +10,64 @@ function noop (err) {
 
 function inject (bot) {
   bot.isSleeping = false
+  const mcData = require('minecraft-data')(bot.version)
+
   const beds = new Set(['white_bed', 'orange_bed', 'magenta_bed', 'light_blue_bed', 'yellow_bed', 'lime_bed', 'pink_bed', 'gray_bed',
     'light_gray_bed', 'cyan_bed', 'purple_bed', 'blue_bed', 'brown_bed', 'green_bed', 'red_bed', 'black_bed', 'bed'])
 
+  const hostileMobs = new Set(['blaze', 'creeper', 'drowned', 'endermite', 'enderman', 'evoker', 'giant', 'guardian', 'elder_guardian',
```

Comment:
Isn't there any way to get the list from the data? I can't imagine updating it with every new minecraft version

Reply:
At first I ussed `entity.kind === 'Hostile mobs`, but it has been removed in 1.13 from minecraft-data, would it be possible to add it again?

Reply:
@rom1504

Reply:
I guess but the only source for that is the minecraft wiki. you can either add manually there or run the extractor that use both wiki.vg and minecraft wiki. (Or add the data to wiki.vg)